// The Firmament project
// Copyright (c) 2013 Malte Schwarzkopf <malte.schwarzkopf@cl.cam.ac.uk>
//

package firmament;

message TaskFinalReport {
  optional uint64 start_time = 1;
  optional uint64 finish_time = 2;
  optional uint64 instructions = 3;
  optional uint64 cycles = 4;
  optional uint64 llc_misses = 5;
  optional double runtime = 6;
}
